piker/clearing/_paper_engine.py

```diff
@@ -286,11 +274,14 @@ async def fake_fill(
             ledger.update(self._trade_ledger)
 
         # Write to pps toml right now
-        with open_pps(self.broker, 'piker-paper') as table: 
+        with open_pps(self.broker, 'piker-paper') as table:
             table.update_from_trans(self._txn_dict)
             # save pps in local state
             self._positions.update(table.pps)
```

Comment:
Yeah in ref to the above comment, and unless maybe i'm missing something, you should be able to just make a `PaperBoi.pp_table: PpTable` instance var and use that directly instead of updating the `._positions: dict`.  I'm not sure if there's some convenience reason for having the dict copy which you have to constantly keep in sync?